### PR TITLE
PIMOB-1394: Fix cornerRadius issue in Constants

### PR DESCRIPTION
**Pull request by patrick-hoban-cko**
## Proposed changes

Small bug fix for cornerRadius.

## Types of changes

What types of changes does your code introduce to Frames Ios?
_Put an `x` in the boxes that apply_

* [ x] Bugfix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: Source/UI/NewUI/CommonUI/View/Core/ButtonView.swift [hunk 1]
```diff
@@ -40,6 +40,7 @@ class ButtonView: UIView {
         self.style = style
         backgroundColor = style.backgroundColor
         layer.cornerRadius = style.cornerRadius
+        button.layer.cornerRadius = style.cornerRadius
```
Comment (deepesh-vasthimal-cko) [inline Source/UI/NewUI/CommonUI/View/Core/ButtonView.swift:43]: Do we have Unit test to update ?

**Comment by ehab-al-cko** [comment]
Happy first PR :) 
please 
- provide Jira ticket to track and test the issue
- screenshot 
- unit test 
- follow git flow for naming repo like **bugfix/PIMOB-xxxx_bug-fix-for-cornerRadius**

**Comment by patrick-hoban-cko** [comment]
> 
Bug here https://checkout.atlassian.net/browse/PIMOB-1394
Screenshot below:
image
Will add unit test
Will update naming

**Comment by patrick-hoban-cko** [comment]
@ehab-al-cko @deepesh-vasthimal-cko please re-review, as there are already tests in place for this change but it didn't work, so changing it wouldn't improve anything. You can see below in `testButtonStyle` on line 22:

image

**Comment by ehab-al-cko** [comment]
@patrick-hoban-cko this issue is fixed in new merge. please pull latest updates and check solution

**Comment by patrick-hoban-cko** [comment]
Closing as @ehab-al-cko has already fixed.

**Pull request closed without merging**